indexer/src/test/java/org/uniprot/store/indexer/uniprotkb/converter/UniProtKBEntryCrossReferenceConverterTest.java

```diff
@@ -93,7 +93,11 @@ void convertGoCrossReferences() {
 
         assertEquals(Set.of("apical dendrite"), document.content);
 
-        assertEquals(new HashSet<>(Arrays.asList("go-GO:12345", "GO:12345")), document.crossRefs);
+        assertEquals(
```

Comment:
Can we filter out if the prefix is equal to database name? this way it would at least filter ("go-GO", "GO"). This would help in the case of GO and also CHEBI, RHEA that I know from the top of my head and they have a lot of xrefs